src/api/ledger/parse/utils.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+const _ = require('lodash');
+const transactionParser = require('ripple-lib-transactionparser');
+const toTimestamp = require('../../../core/utils').toTimestamp;
```

Comment:
I added this https://www.npmjs.com/package/root-require to ripple-connect to avoid the `../../../`